setup.cfg

```diff
@@ -112,6 +112,7 @@ markers =
     remote_data: marks this test function as needing remote data.
     online: marks this test function as needing online connectivity.
     mpl_image_compare: marks this test function as using hash-based Matplotlib figure verification. This mark is not meant to be directly applied, but is instead automatically applied when a test function uses the @sunpy.tests.helpers.figure_test decorator.
+    remote_status: mark this test function as reporting back status of remote requests.
```

Comment:
Reports back in the test output I assume?

Reply:
Yes